### ISSUE TITLE
Chrome 75 changes the path to the KeystoneRegistration.framework

### ISSUE THREAD

**Issue by sdpalmer** [hjuutilainen/adminscripts#14]
The correct path is now /Applications/Google Chrome.app/Contents/Frameworks/Google Chrome Framework.framework/Frameworks/KeystoneRegistration.framework.

I don't know python as well, but I was able to figure out that I needed to change the keystone_registration_framework_path() subroutine to the following:
```
def keystone_registration_framework_path():
    """Returns KeystoneRegistration.framework path"""
    keystone_registration = os.path.join(chrome_path, 'Contents/Frameworks/Google Chrome Framework.framework/Frameworks/KeystoneRegistration.framework')
    return keystone_registration
```

I don't know enough, yet, to figure out how to make it work for both versions earlier than 75 and version 75 or later.

**Comment by ryangball** [comment]
@sdpalmer This was corrected using the latest push, no?

**Comment by sdpalmer** [comment]
@ryangball Yes, you are correct.  I had checked late last week and didn't see any changes and checked again earlier today before figuring what the cause was.  However, this morning I only looked at the version history at the top of the script and seeing no changes listed I assumed there were no changes specific to Chrome 75.
